yarn.lock

```diff
@@ -4062,10 +4062,25 @@ typedoc@^0.25.7:
     minimatch "^9.0.3"
     shiki "^0.14.7"
 
-typescript@4.6.3:
```

Comment:
This doesn't seem to be the matching yarn.lock file for this `typescript": "5.1.6"` update.

Reply:
This issue happened while rebasing, `yarn.lock` had too many conflicts to and it was not clear which version to go with.
This conflicts rose after https://github.com/evolvedbinary/prosemirror-jdita/pull/241 was merged into main.
I made one commit to address the `yarn.lock` issue: [ee8a465](https://github.com/evolvedbinary/prosemirror-jdita/pull/233/commits/ee8a4659d01aea5070014e056c1cfbf1667751f1)